course/lesson-5-your-first-function/lesson.tres

```diff
@@ -39,9 +39,9 @@ text = "In computer programming, we talk about [i]identifiers[/i] rather than \"
 
 It is because a function name is a label the computer uses to precisely [i]identify[/i] and refer to a function or other code elements.
 
-Identifiers are unique: you cannot reuse the same name in a given [i]space[/i] in your code.
+Identifiers are unique: you cannot reuse the same name in a given [i]script[/i] in your code.
```

Comment:
People don't know about scripts here so we shouldn't talk about scripts. More importantly, talking about scripts is incorrect: you can reuse identifiers in the same script in different code files - that's what "space" is supposed to hint at (a function, a code block, an inner class etc.).